CONTRIBUTING.md

```diff
@@ -49,7 +49,12 @@ If you have an idea for a query that you would like to share with other CodeQL u
 
     - The query must have at least one true positive result on some revision of a real project.
 
-Experimental queries and libraries may not be actively maintained as the [supported](docs/supported-queries.md) libraries evolve. They may also be changed in backwards-incompatible ways or may be removed entirely in the future without deprecation warnings.
+6. **Query Help Files and Unit Tests**
```

Comment:
```suggestion
6. **Query help files and unit tests**
```

We use sentence case rather than title case for headings in this file and, I believe, most other CodeQL docs.

Reply:
Thanks.